tests/integrationv2/providers.py

```diff
@@ -54,22 +93,22 @@ def __init__(self, options: ProviderOptions):
         Provider.__init__(self, options)
 
     def setup_client(self, options: ProviderOptions):
-        self.ready_to_test = 'listening on lo'
-        tcpdump_filter = f"dst port {options.port}"
+        self.ready_to_test_marker = 'listening on lo'
+        tcpdump_filter = "dst port {}".format(options.port)
 
         cmd_line = ["tcpdump",
             # Line buffer the output
             "-l",
 
-            # Only read 12 packets before exiting. This is enough to find a large
+            # Only read 10 packets before exiting. This is enough to find a large
             # packet, and still exit before the timeout.
-            "-c", "12",
+            "-c", "10",
```

Comment:
was there a reason to decrement from 12 to 10?

Reply:
There was. We always send at least 10 packets. Sometimes we don't send more than that. In those cases we would timeout from time to time. Checking for 10 packets is consistent.